examples/minikube/README.rst

```diff
@@ -5,12 +5,15 @@ Deploying Cilium on a minikube cluster is simple. The provided
 ``cilium-ds.yaml`` file contains a DaemonSet spec which can be used to take
 care of all deployments steps.
 
-Deploy a minikube cluster using the ISO as described below and instruct
-Kubernetes to use the CNI plugin infrastructure:
+Deploy a minikube cluster as described bellow to instruct Kubernetes to use the
+CNI plugin infrastructure plus the ``localkube`` bootstrapper since it contains
+``etcd`` >= ``3.1.0`` required by Cilium. The ``CustomResourceValidation`` will
+allow Cilium to install the Cilium Network Policy validator into kubernetes
+(`more info <https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#validation>`_).
```

Comment:
This paragraph is a copy of the one in gettingstarted/minikube_intro.rst If you accept the edit I had there, it would apply here too.

Reply:
Done